### ISSUE TITLE
Exposure to raw non-synthetic textiles

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10038]
**Preferred term label:**
Exposure to raw non-synthetic textiles

**Synonyms**
cotton

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C0239130

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Byssinosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000449
- label: Occupational exposure to raw nonsynthetic textiles
- parents: [Occupational exposure history (HP:0035010)](https://hpo.jax.org/app/browse/term/HP:0035010)
- pmid: PMID:30137833
- def: A history of exposure by inhalation of airborne particles generated during the processing of raw, nonsynthetic textiles, particularly cotton. Classically, exposure to cotton dust during the spinning and manufacturing process causes byssinosis. However, exposure to jute, flax, and hemp fibers has also been implicated in its development.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10038
- synonyms:
